http/src/main/resources/reference.conf

```diff
@@ -493,7 +493,7 @@ gke {
 image {
   welderGcrUri = "us.gcr.io/broad-dsp-gcr-public/welder-server"
   welderDockerHubUri = "broadinstitute/welder-server"
-  welderHash = "342aff2"
+  welderHash = "2bcce86"
```

Comment:
this needs to be updated to the hash once welder PR is merged

Reply:
correct welder hash was merged in this PR https://github.com/DataBiosphere/leonardo/pull/2613